hw3/tests/test_hw3_task01.py

```diff
@@ -0,0 +1,45 @@
+from hw3.hw3_task01 import TestPlug, cache_factory
```

Comment:
Why Test plug is not in tests?  

Why do you need to create this plug if you can use Mock() instead?

Reply:
It doesn't work if it is not in the main file. I'll try to mock it.